python/cudf/cudf/tests/test_rolling.py

```diff
@@ -310,10 +300,8 @@ def test_rolling_getitem():
 
 
 def test_rolling_getitem_window():
-    if PANDAS_GE_110:
-        kwargs = {"check_freq": False}
-    else:
-        kwargs = {}
+    kwargs = {"check_freq": False}
```

Comment:
Let's explicitly specify these in the function instead of writing `**kwargs`. It's specific to cudf and isn't used in the pandas call. This applies in a couple places.

Reply:
Agreed, these were previously `kwargs` to avoid polluting the `assert_eq` with pandas version conditions, but now those are gone we can just inline things.

Reply:
Done